include/edyn/comp/continuous.hpp

```diff
@@ -18,14 +18,14 @@ struct continuous {
 
     template<typename... Component>
     void insert() {
-        ((types[size++] = entt::type_index<Component>::value()), ...);
+        ((types[size++] = entt::type_id<Component>().hash()), ...);
```

Comment:
I think we should be using [`entt::type_id<T>().seq()`](https://github.com/skypjack/entt/wiki/Crash-Course:-core-functionalities#type-info) here which is the actual replacement for `entt::type_index<T>::value()`.